pages/partials/navbar.ejs

```diff
@@ -124,3 +125,18 @@ if (typeof authz_data !== 'undefined' && authz_data.has_instructor_view) {
 </ul>
 </div>
 </nav>
+<% // Mapping navPage to navtab sets
+const navPagesTabs = {
+    instance_admin: 'instructorInstanceAdminTabs',
+    course_admin: 'instructorCourseAdminTabs',
+    assessment: 'instructorAssessmentTabs',
+    question: 'instructorQuestionTabs'
+};
+// If navPage matches any of the keys in navPagesTabs, include the page given by its value
+if (typeof navPage != 'undefined' && Object.keys(navPagesTabs).includes(navPage)) { %>
+    <!-- Instructor navigation tabs -->
+    <ul class="nav nav-tabs pt-2 px-3 bg-light">
```

Comment:
I think this is missing an `id=...` attribute. 

e.g. `id="navtabsInstructorQuestion"`

Not sure if this is needed though.

Reply:
I did a scan through the codebase for the IDs that each of the instructor navtabs files specified, but didn't find any reference to them anywhere else, so I thought it best to delete them.